doc/examples.rst

```diff
@@ -76,3 +76,80 @@ Or just one of them:
 
    $ tt start demo:master
    • Starting an instance [master]...
+
+Working with application templates
+----------------------------------
+
+For example, we want to create an application template. In order to do this, create
+a directory, which will be used as a templates storage - ``./templates``. Then
+create an application template directory in it: ``./templates/simple`` with
+the content:
+
+``init.lua.tt.template``:
+
+.. code-block:: lua
+
+    local app_name = {{.name}}
+    local login = {{.user_name}}
+
+    require("fiber").sleep(1)
+
+``MANIFEST.yaml``:
+
+.. code-block:: yaml
+
+    description: Simple app
+        vars:
+            - prompt: User name
+            name: user_name
+            default: admin
+            re: ^\w+$
+
+Create a symlink in the template directory:
+``{{.name}} -> init.lua``
+
+Directory name ``simple`` can be used as a template name in create command.
+Add templates search path to ``./tarantool.yaml`` config:
+
+.. code-block:: yaml
+
+    tt:
+        templates:
+            -path: ./templates
+
+Here is how the directory structure looks like::
+
+    ./
+    ├── tarantool.yaml
+    └── templates
+        └── simple
+            ├── init.lua.tt.template
+            ├── MANIFEST.yaml
+            └── {{.name}}.lua -> ./init.lua
```

Comment:
it is not clear what the init.lua file contains, where is the description?

Reply:
Init.lua is just a sample of app code.